### ISSUE TITLE
Add Monero Address

### ISSUE THREAD

**Issue by SamsungGalaxyPlayer** [keybase/client#6208]
Apologies if this is in the wrong repository. I will repost if directed.

I would like to see support for a Monero address in the Keybase client. Monero is a private, fungible cryptocurrency that has been around since 2014. 

According to [this blog post](https://keybase.io/blog/keybase-and-zcash), the following is what Keybase is looking for in a cryptocurrency:

> Zcash fixes the privacy issues, while retaining the usability we want: just post an address on your profile once, and that's it. Let the money flow in.

Monero has the same functionality. One can post *any* Monero address, and it is private by default. If you would also like to allow visibility (in relation to "tip jars or donations"), you can also ask users for an optional view key that will show the incoming transactions.

Monero has a substantially larger market capitalization than Zcash, and it has a larger community. I believe if Keybase found enough interest for Zcash, there is also enough interest for Monero.

Learn more at getmonero.org. If you are having difficulty adding Monero, please visit [the Reddit page](www.reddit.com/r/monero) or #monero-dev on IRC.

**Comment by tommorris** [comment]
Other cryptocurrency issues: #5788 (reusable payment codes), #7042 (altcoins), #7560 and #4266 (Ethereum).

**Comment by SamsungGalaxyPlayer** [comment]
Zcash is already added as an altcoin, so I see no reason other coins should not be supported.

**Comment by rubik** [comment]
I second this request. Is there something blocking this issue, something we might help with?

**Comment by Electricsheep01** [comment]
Bump - is this being looked at?

**Comment by cialu** [comment]
Monero is secure and private, probably the best crypto for keybase purposes.

I join this request.

**Comment by xamox** [comment]
I concur.

**Comment by rafassdotcom** [comment]
Monero xmr is probably the most robust and solid privacy coin actually.

**Comment by Kr4ken-9** [comment]
Bump - There is still interest in this request.

**Comment by InnovativeInventor** [comment]
Agreed, whats the status on this? Lots of people really want this feature.

**Comment by SamsungGalaxyPlayer** [comment]
If you want to be extremely innovative, you can allow only Monero subaddresses to be posted :)

**Comment by mutagenfork** [comment]
Adding my voice to the chorus.  Please add Monero support!

**Comment by umma08** [comment]
keybase have added ZCash and Grin before adding Monero (even adding Grin while the protocol is going through inevitable growing pains). 

There must be some reason why Monero is not added, and i would hazard a guess the reason is not entirely technical.

**Comment by hyc** [comment]
Would be nice to get even a response then, if there's some underlying issue blocking this.

**Comment by fire-pig** [comment]
Yes, please add Monero support!  Or at least let us know what the blocking issues are.  Thanks!

**Comment by maitscha** [comment]
Please add Monero!

**Comment by mgrube** [comment]
I use keybase almost daily and would really appreciate monero support. Thank you.

**Comment by PaulSpoerry** [comment]
XMR FTW!

**Comment by duckrocks** [comment]
I think the problem with adding Monero support is that they have a sort of aggressive and fanatical community that oftentimes goes out of their way to attack other coins. It may be created some "bad blood" between other coins to add Monero. ZCash is relatively benign in that regard. At least imo.

**Comment by rehrar** [comment]
@duckrocks let me provide an alternative take on this.

The Monero community is very passionate about privacy, and we all acknowledge that real privacy is incredibly hard to get right. Those of us in the Monero community even realize that Monero has several weaknesses in regards to our own privacy.

What is very frustrating is when coins with gigantic claims to fame about their "impenetrable privacy" pop up out of the wood work all the time. Projects like Verge, TokenPay, and more. These projects usually offer incredibly weak privacy at best, and their "marketing claims" (that are just trying to make them a quick buck) are very dangerous to anyone who believes them.

The Monero community tends to refute these claims, swiftly and without apology, and in so doing, Monero has gotten a (hilarious) reputation amongst other coins as being "trolls" and "party poopers", just because Monero has the audacity to rain on the parades of the moon kids by telling them that their privacy isn't up to snuff.

It'd be similar to someone rolling their own crypto (noticeably and verifiably badly) and putting it in a chat app and claiming that you'll be "safe and encrypted forevermore" and something like Signal comes in and says "no, that's not how encryption works" at which point all the people working on the theoretical app (or otherwise invested) say that Signal goes out of their way to be aggressive and fanatical to other encryption apps because it wants to stay on top.

There has indeed been some bad blood at times between Monero and Zcash, although many in the community acknowledge our goals are more aligned than many might think, it's just our implementations and ideals are drastically different. This was clearly evidenced by the cross-country privacy panel that occurred when MoneroKon and Zcon1 were taking place at the same time on different parts of the world: https://www.youtube.com/watch?v=DWPELgmS2uA

In summary, while Monero can often times be passionate about their approach to correcting bad information, I think its erring into hyperbole to say the majority of the community is aggressive and fanatical.

**Comment by duckrocks** [comment]
Ah ok, I understand. I do not mean to insult the entire Monero community, obviously you do good work. What I was merely getting at was there have been more than one occasion where I or a colleague have done research on various cryptocurrency related topics, and when coming in the privacy sphere seeing some, I don't know how else to put it, "bad behavior". 

And this was usually from **certain, select** members of the Monero community, and yes it was usually in relation to the privacy of other coins. I think its fair to put it down to zeal or passion or whatever, but all I meant was that I can see how it could possibly make it politically more difficult to add XMR on a platform because of this. Some may see it and be turned off from the coin even though it was merely the behavior of a few loose cannons and zealous guys.

This is all I say, and it is just my limited opinion. I do not mean to come across as attacking you or your coin.

**Comment by jwinterm** [comment]
Would be nice, especially if it actually supported a wallet, not just address. I guess stellar paid a bunch of money to get added? How much cash does it take? I got about tree fiddy.

**Comment by deadblackclover** [comment]
If you add Monero, it would be very cool!

**Comment by artstr1** [comment]
Add XMR! ;)

**Comment by johnalanwoods** [comment]
Monero is definitely important to add.

**Comment by CryptAncapitalist** [comment]
I agree, Monero (XMR) should be added to Keybase ASAP, as it is the closest coin to Satoshi's true vision of 1-CPU, 1-vote due to its ASIC resistance, and it is also uniquely fungible due to its enforced privacy by default, unlike Bitcoin and the vast majority of altcoins.

**Comment by aboardnow60** [comment]
Please add Monero support!

**Comment by tiedtoastar** [comment]
How can the monero community help to get this going?

**Comment by invertedbobb** [comment]
Best fit considering the nature and solid development of both projects.
Love to see this happen.

**Comment by mister-user** [comment]
upvote

**Comment by damienbutt** [comment]
+1 👍👍
